docs/examples/index.md

```diff
@@ -17,6 +17,23 @@ Reference Applications are fully documented, end-to-end solutions that show how
 * [Background Check Application in Go](go/background-checks/index.md)
 * [Order Management System in Go](https://github.com/temporalio/reference-app-orders-go)
 
+# Example Applications
+While not intended for production use, Example Applications can nonetheless be useful for seeing Temporal in action for particular use cases and architectural design patterns.
+
+## Use cases
+* **Order Fulfillment** [[TypeScript]](https://github.com/temporal-sa/temporal-order-fulfill-demo) A Temporal workflow for a sample e-commerce order fulfillment use case. This demo showcases Temporal's powerful durability and interactive capabilities. See companion [video](https://www.youtube.com/watch?v=dNVmRfWsNkM).
+* **Money Transfer** [[Java]](https://github.com/temporal-sa/temporal-money-transfer-java) [[TypeScript]](https://github.com/temporal-sa/temporal-money-transfer-typescript) demonstrates core value propositions of Temporal. A custom UI allows users to choose from a number of simulated failure scenarios and use of Temporal primitives such as signals and schedules.
+* **Orchestrate Lambda Functions** [[TypeScript]](https://github.com/temporal-sa/temporal-orchestrate-lambda-functions) A Temporal version of the [AWS Step Functions: Lambda orchestration example](https://docs.aws.amazon.com/step-functions/latest/dg/sample-lambda-orchestration.html) which uses Lambda functions to check a stock price and determine a buy or sell trading recommendation.
+
+## Design Patterns
+* **Entity Workflow Demo** [[Go]](https://github.com/temporal-sa/temporal-entity-lifecycle-go) The purpose of this demo is to illustrate some of the interesting properties of the Entity Lifecycle Pattern (aka Entity Workflows). See companion slide deck on [Long-running workflows](https://docs.google.com/presentation/d/1A2dz4lFiIFz4c_7QlOpahbvesbBY8Y6y65zRrkVgqYE/edit?usp=sharing).
+* **Order Saga Sample** [[Java]](https://github.com/temporal-sa/temporal-order-saga) Demonstrates Temporal's enhanced support for the Saga pattern, supporting recovery from failed transactions. See companion [video](https://www.youtube.com/watch?v=uHDQMfOMFD4).
+
+## Data Encryption
+See the [Temporal Platform Feature Guide](https://docs.temporal.io/production-deployment/data-encryption) for more information on the purpose of Temporal Codec Servers.
```

Comment:
```suggestion
See the [Codec Server documentation](https://docs.temporal.io/production-deployment/data-encryption) for more information.
```

Reply:
Tagging in @steveandroulakis on this one, since it was you who added that link + description I believe. Are you cool with this re-wording?

Reply:
Thanks @napcs your wording is way clearer and should be adopted.